Code/GraphMol/Chirality.h

```diff
@@ -55,11 +55,69 @@ RDKIT_GRAPHMOL_EXPORT const Bond *getNeighboringDirectedBond(const ROMol &mol,
  */
 RDKIT_GRAPHMOL_EXPORT Bond::BondStereo translateEZLabelToCisTrans(
     Bond::BondStereo label);
+/// @endcond
+
+enum class StereoType {
+  Unspecified,
+  Atom_Tetrahedral,
+  Bond_Double,         // single double bond and odd-numbered cumulenes
+  Bond_Cumulene_Even,  // even-numbered cumulenes
+  Bond_Atropisomer
+};
+
+enum class StereoDescriptor { None, Tet_CW, Tet_CCW, Bond_Cis, Bond_Trans };
+
+enum class StereoSpecified {
+  Unspecified,  // no information provided
+  Specified,
+  Unknown  // deliberately marked as unknown
+};
+
+struct RDKIT_GRAPHMOL_EXPORT StereoInfo {
+  // REVIEW: absolute stereo data member?
+#ifdef _MSC_VER
+  static const unsigned NOATOM =
+      std::numeric_limits<unsigned>::max();  // used to mark missing atoms
+#else
+  static const unsigned NOATOM;  // used to mark missing atoms
+#endif
+  StereoType type = StereoType::Unspecified;
+  StereoSpecified specified = StereoSpecified::Unspecified;
+  unsigned centeredOn = NOATOM;
+  StereoDescriptor descriptor = StereoDescriptor::None;
+  std::vector<unsigned> controllingAtoms;  // all atoms around the atom or bond.
+  // Order is important
+  bool operator==(const StereoInfo &other) const {
+    return type == other.type && specified == other.specified &&
+           centeredOn == other.centeredOn && descriptor == other.descriptor &&
+           controllingAtoms == other.controllingAtoms;
+  }
+};
+
+/*!
+  \param mol the molecule to look for stereo int
```

Comment:
there's a typo here too.
I agree that it would be useful to document what we consider to be stereogenic atoms and bonds.
I will add something to the RDKit book for this.